Automattic/sensei
# Rename "View Quiz" block to "Take Quiz" block

## Proposed Changes
Renames the _View Quiz_ block to _Take Quiz_ block in order to match the button text, as it's a bit confusing trying to find the block in the editor's list view. This PR doesn't update the block name in order to preserve backward compatibility.

Note that one issue with this change is that for non-Learning Mode lessons, the quiz button will still say _View Quiz_ until the lesson is resaved.

## Testing Instructions


1. Enable Learning Mode.
2. In the Lesson template, check that the button says _Take Quiz_ as well as the block in List View.
3. Open a lesson in the editor and check that the block is named _Take Quiz_ in List View.
4. View a lesson containing the Lesson Actions block and a quiz on the frontend.
5. Ensure the button says _Take Quiz_.
6. Disable Learning Mode.
7. Open the lesson in the editor and check that the button says _Take Quiz_.
8. Save the lesson.
9. View the lesson on the frontend and ensure that buttons says _Take Quiz_.
10. No lessons or templates should break with this change.

## Screenshot
![Screenshot 2023-11-27 at 4 53 31 PM](https://github.com/Automattic/sensei/assets/1190420/646e379f-d5c9-4344-9b99-e8f650751014)

## Pre-Merge Checklist
